examples/chart/teleport-cluster/charts/teleport-operator/templates/resources.teleport.dev_roles.yaml

```diff
@@ -859,8 +859,12 @@ spec:
                       users what they aught to
                     type: string
                   require_session_mfa:
-                    description: RequireSessionMFA specifies whether a user is required
```

Comment:
@marcoandredinis I ran `make -C operator manifests` to update this. Are there backwards compatiblity concerns or other possible issues to consider here?

Reply:
Also, it looks like this didn't fix my issue. I'm still setting this error in the unit tests:
```
--- FAIL: TestRoleCreation (5.69s)
    suite_test.go:123: 
        	Error Trace:	suite_test.go:123
        	            				suite_test.go:217
        	            				role_controller_test.go:44
        	Error:      	Received unexpected error:
        	            	unable to install CRDs onto control plane: unable to create CRD instances: unable to create CRD "teleportroles.resources.teleport.dev": CustomResourceDefinition.apiextensions.k8s.io "teleportroles.resources.teleport.dev" is invalid: spec.validation.openAPIV3Schema.properties[spec].properties[options].properties[require_session_mfa].type: Required value: must not be empty for specified object fields
        	Test:       	TestRoleCreation
```

What do I need to do to update the Kuberenetes Operator for this PR?

Reply:
Turns out that enums were not supported by `protoc-gen-crd`. Fixed in 0ac1174631e9d2488e642a3551126dc79e697b39.

Reply:
It seems to be working as expected after your fix
Thank you :bow: